server/container_create_linux.go

```diff
@@ -495,6 +495,7 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID, contai
 				return nil, fmt.Errorf("set memory limit %v too low; should be at least %v", memoryLimit, minMemoryLimit)
 			}
 			specgen.SetLinuxResourcesMemoryLimit(memoryLimit)
+			specgen.SetLinuxResourcesMemorySwap(memoryLimit)
```

Comment:
a work around for kernels not having this would be checking if `/sys/fs/cgroup/memory/memory.memsw.limit_in_bytes` exists before setting this. I would think this would be a responsibility of the runtime though...